charts/rancher-monitoring/v0.0.4/charts/operator-init/templates/job-install-crds.yaml

```diff
@@ -16,6 +16,10 @@ spec:
       tolerations:
 {{- include "linux-node-tolerations" . | nindent 8}}
       serviceAccountName: {{ template "app.fullname" . }}
+      securityContext:
+        runAsUser: 1000
+        runAsNonRoot: true
+        fsGroup: 2000
```

Comment:
UID 1000 and GID 2000 do not exist in /etc/passwd. let's use 65534(nobody). See https://github.com/helm/charts/blob/478441f4ba2ddfd55ebf3139f9c5f85fb462efa5/stable/kube-state-metrics/values.yaml#L59-L62

Reply:
Thanks, updated the PR